strict.js

```diff
@@ -43,13 +43,13 @@ module.exports = {
 		'no-array-constructor': 'error',
 		'no-cond-assign': ['error', 'except-parens'],
 		'no-console': 'warn',
-		'no-debugger': 'warn',
+		'no-debugger': 'error',
```

Comment:
If these `warn` to `error` rules in the eslint:recommended, can we just remove them?

Reply:
OK, let's remove it.
Not exactly, but before eslint 6.0.0(before May 2019), `eslint:recommend` already set 'no-debugger' to 'error'.
(https://github.com/eslint/eslint/commit/f42d0afd89874b459fce1eb1998247d53f9aa42b)

Reply:
OK, let's remove it.
Not exactly, but before eslint 6.0.0(before May 2019), `eslint:recommend` already set 'no-debugger' to 'error'.
(https://github.com/eslint/eslint/commit/f42d0afd89874b459fce1eb1998247d53f9aa42b)